skipruntime-ts/server/src/rest.ts

```diff
@@ -1,13 +1,41 @@
 import express from "express";
-import crypto from "crypto";
-import type { Entry, Json, ServiceInstance, CollectionUpdate } from "skip-wasm";
 import { UnknownCollectionError } from "skip-wasm";
+import type { Entry, Json, ServiceInstance, CollectionUpdate } from "skip-wasm";
 
-export function createRESTServer(service: ServiceInstance): express.Express {
+export function controlService(service: ServiceInstance): express.Express {
   const app = express();
   app.use(express.json());
+
+  // Streaming control API.
+  app.post("/v1/streams", (req, res) => {
+    try {
+      const uuid = crypto.randomUUID();
+      service.instantiateResource(
+        uuid,
+        req.body.resource as string,
+        req.body.params as { [param: string]: string },
+      );
+      res.status(201).location(`/v1/streams/${uuid}`).json(uuid);
+    } catch (e: unknown) {
+      res.status(500).json(e instanceof Error ? e.message : e);
+    }
+  });
+
+  app.delete("/v1/streams/:uuid", (req, res) => {
+    try {
+      service.closeResourceInstance(req.params.uuid);
+      res.sendStatus(200);
+    } catch (e: unknown) {
+      res.status(500).json(e instanceof Error ? e.message : e);
+    }
+  });
+
+  app.get("/v1/streams/:uuid", (_req, res) => {
+    res.status(200).json("TODO");
```

Comment:
?

Reply:
This is supposed to be a "maintenance"/"debug" endpoint, responding with data about the streaming session itself. As it is not necessary for now, I'll remove the endpoint from this PR altogether.